src/comgr.cpp

```diff
@@ -1055,27 +1054,23 @@ struct Error : std::exception
     throw Error{s, text};
 }
 
-static inline std::string to_string(const std::string& v) { return {v}; }
-static inline std::string to_string(const char* v) { return {v}; }
-static inline auto to_string(const std::size_t& v) { return std::to_string(v); }
```

Comment:
Please revert this. `HIPRTC_CALL_BASE` macro accepts `info` of any type. I want `info` to have the `to_string()` converter as a spare level of protection.

Reply:
The code formats a message using a stream object. That is much safer and more versatile than the `to_string()` functions mentioned. I see those functions as redundant. Can you please elaborate more on the level of protection you mentioned?

Reply:
I checked, and on Windows, we are getting here: `C:\...\MIOpen\src\comgr.cpp:1115:5: error: no matching function for call to 'to_string'`. That is the reason for the change.

Reply:
[resolved]